docs/_documentations/metrics-dashboard.md

```diff
@@ -29,7 +29,7 @@ Click **Metrics Dashboard** in the Codewind project to access the Metrics Dashbo
 
 ## Enabling Application Metrics
 
-If you import a project that does not include Application Metrics monitoring data associated with it, you can add Application Metrics by including the appropriate resource or performance monitoring application. For more information, see [RuntimeTools](https://github.com/RuntimeTools/).
+If you import a project that does not include Application Metrics monitoring data associated with it, you can add Application Metrics by including the appropriate resource or performance monitoring application. For more information, see [RuntimeTools](https://github.com/RuntimeTools/). For more information about Application Metrics, see [Application Metrics on GitHub](https://github.com/RuntimeTools/appmetrics).
```

Comment:
I think the reason we used the `RuntimeTools` link is because `RuntimeTools/appmetrics` is just for NodeJS. I'm not sure if it's the right thing to link to here.

Reply:
Removed